modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -98,4 +100,18 @@ export class Lightning implements ILightning {
       throw new Error(`error(s) parsing response body: ${errors}`);
     });
   }
+
+  public async getInvoices(query?: GetInvoicesQuery): Promise<GetInvoicesResponse> {
+    const queryParams: { status?: string; limit?: number } = {};
+    queryParams.status = query?.status;
+    queryParams.limit = query?.limit;
+
+    const body = await this.bitgo
+      .get(this.url + '/invoices')
+      .query(queryParams)
+      .result();
+    return decodeOrElse(GetInvoicesResponse.name, GetInvoicesResponse, body, (errors) => {
+      throw new Error(`error(s) parsing response body: ${errors}`);
+    });
```

Comment:
can we make this more generic

```suggestion
    return this.request('get', '/invoices', queryParams, GetInvoicesResponse.name);
```

we will save a lot of boilerplate that way

(can happen in next PR)